### PR TITLE
e2e_gcs test scenarios

### PR DESCRIPTION
**Pull request by itsmekumari**
GCS e2e Tests

### CODE REVIEW

File: src/e2e-test/features/gcs/sink/GCSSink.feature [hunk 1]
```diff
@@ -199,3 +199,69 @@ Feature: GCS sink - Verification of GCS Sink plugin
...
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Verify data is transferred to target GCS bucket
+    Then Validate the values of records transferred to GCS bucket is equal to the values from source BigQuery table
+
+  @BQ_SOURCE_TEST @GCS_SINK_TEST
+  Scenario: To verify data is getting transferred successfully from BigQuery to GCS using delimiter format
```
Comment (itsankit-google) [inline src/e2e-test/features/gcs/sink/GCSSink.feature]: nit: `To verify data is getting transferred successfully from BigQuery to GCS using delimiter output format`

File: src/e2e-test/features/gcs/sink/GCSSink.feature [hunk 2]
```diff
@@ -199,3 +199,69 @@ Feature: GCS sink - Verification of GCS Sink plugin
     Then Open and capture logs
     Then Verify the pipeline status is "Succeeded"
     Then Verify data is transferred to target GCS bucket
+
+  @BQ_SOURCE_DATATYPE_TEST @GCS_SINK_TEST
+  Scenario: To verify data is getting transferred successfully from BigQuery to GCS with supported datatypes
```
Comment (itsankit-google) [inline src/e2e-test/features/gcs/sink/GCSSink.feature]: nit: `To verify data is getting transferred successfully from BigQuery to GCS using json output format`

**Review by itsankit-google — COMMENTED** [review]
Please look into test failures.

File: src/e2e-test/features/gcs/sink/GCSSink.feature (hunk 1 above)
Comment (itsmekumari) [reply src/e2e-test/features/gcs/sink/GCSSink.feature]: Change done.

File: src/e2e-test/features/gcs/sink/GCSSink.feature (hunk 2 above)
Comment (itsmekumari) [reply src/e2e-test/features/gcs/sink/GCSSink.feature]: Change done.

**Comment by sau42shri** [comment]
@itsmekumari Can you fix the build and I will ask @itsankit-google to do the final review.

**Comment by itsmekumari** [comment]
> @itsmekumari Can you fix the build and I will ask @itsankit-google to do the final review.

Verified the report, a few tests are failing due to an intermittent chrome version issue, and we have reported a bug. https://cdap.atlassian.net/browse/PLUGIN-1649

**Review by itsankit-google — APPROVED** [review]

**Pull request merged**